build/dockerfiles/Dockerfile

```diff
@@ -10,39 +10,8 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-# Builder: check meta.yamls and create index.json
-FROM docker.io/node:12.19.0-alpine3.12 AS builder
-RUN apk add --no-cache py3-pip jq bash git wget skopeo && pip3 install yq jsonschema
-
```

Comment:
If we no longer use multistage build, it would required the right dependencies to build locally. Now - nodejs 12.x and yarn v1 - I don't see the value removing it.

Reply:
> it would required the right dependencies to build locally

it is not what the devfile provide ? ready-to-go environment ?

Reply:
yeah ok so we should remove anything mentioning how to build "locally". Contributors should only use Che.

Reply:
also ... how can we unsure that what works in the devfile would work in ci if we don't use the same environment. For instance, I could use `skopeo` in my build environment but it would fail because it is not in the CI environment.
The container used in the devfile to build should be the one used in our CI.

Reply:
we no longer use `skopeo`, it only requires nodejs

Reply:
once we remove tree utility it means we only need nodejs runtime (no need to have che-plugin-dev) image

Once it's set, then a nodejs image could be used without problems by CI, by Che or by people wanted to run it on their local computer or use their existing nodejs runtime.

And we can tweak build.sh to have node command being run inside a container or not if you prefer.

In any case we're splitting previous multi-stage build in two (but as first stage should be an existing community node image then we keep only the later stage as part of this Dockerfile)